Lib/typing.py

```diff
@@ -1,20 +1,21 @@
 """
-The typing module: Support for gradual typing as defined by PEP 484.
+The typing module: Support for gradual typing as defined by PEP 484 and subsequent PEPs.
 
 At large scale, the structure of the module is following:
 * Imports and exports, all public names should be explicitly added to __all__.
 * Internal helper functions: these should never be used in code outside this module.
 * _SpecialForm and its instances (special forms):
```

Comment:
I don't think we should be mentioning at all whether these things are instances of `_SpecialForm`. Whether they use `_SpecialForm` or some other implementation mechanism is an implementation detail that no users should have to care about.